.circleci/config.yml

```diff
@@ -165,6 +166,10 @@ workflows:
           # container_gun_image_name: 'openjdk'
           # container_gun_image_tag: '11.0.3-jdk-stretch'
           container_size: 'large'
+      - apim/d_javascript_package_bundle:
```

Comment:
I don't understand why we have a dedicated step here, I think it should be the same for all policies